styles/global.css

```diff
@@ -76,20 +119,65 @@ body {
 }
 
 .dropdown-content {
-  background-color: rgb(253, 252, 252);
+  background-color: black;
+  /* hide dropdown content by default */
   display: none;
   width: 100px;
+  /* make content appear below dropdown button */
   position: absolute;
+  top: 100%;
 }
 
 .dropdown-content a {
   /* start with new line */
   display: block;
-  color: black;
+  color: white;
+  padding: 10px;
 }
 .dropdown:hover .dropdown-content {
   /* without this, display: none for content */
   display: block;
   padding: 12px;
   text-align: left;
 }
+.search-container {
+  border-radius: 4px;
+  padding-left: 6px;
+  padding-right: 6px;
```

Comment:
Why the padding here is smaller than the rest and the content has margin left?
It should still work with padding-left/right 20px like the rest of the items

Reply:
yes, I removed the margin-left and add padding 20px 20px same as the rest